public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,131 @@
+# dvc.api.open()
+
+Opens a file <abbr>artifact</abbr> as a
+[file object](https://docs.python.org/3/glossary.html#term-file-object). May
+only be used as
+[context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library).
+
+> This has similar uses as the `dvc get` and `dvc import` CLI commands.
+
+💡 Note that `dvc.api.open` is able to
+[stream](https://docs.python.org/3/library/io.html) the file directly from
+**most**
+[remote types](/doc/command-reference/remote/add#supported-storage-types)
+(local, S3, Azure, GCP, OSS, SSH, HDFS). Otherwise, the file is downloaded
+regularly into a temporary local path before the file object is made available.
```

Comment:
@efiop is this an appropriate link for the word "stream"? Thanks

Reply:
Yes, @jorgeorpinel.